4-WebApp-your-API/4-2-B2C/Client/appsettings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "AzureAdB2C": {
+    "Instance": "https://fabrikamb2c.b2clogin.com",
+    "ClientId": "fdb91ff5-5ce6-41f3-bdbd-8267c817015d",
+    "Domain": "fabrikamb2c.onmicrosoft.com",
+    "SignedOutCallbackPath": "/signout/B2C_1_susi",
+    "SignUpSignInPolicyId": "b2c_1_susi",
+    "ResetPasswordPolicyId": "b2c_1_reset",
+    "EditProfilePolicyId": "b2c_1_edit_profile", // Optional profile editing policy
+    "ClientSecret": "X330F3#92!z614M4"
+    //"CallbackPath": "/signin/B2C_1_sign_up_in"  // defaults to /signin-oidc
+  },
+  "AzureAd": {
```

Comment:
Why do we need AzureAD section in this B2C appsettings?

Reply:
We don't. This is merged in RemoveUIs, we should remove it from there.